packages/mwp-api-state/src/mutate/index.test.js

```diff
@@ -11,7 +11,7 @@ describe('postEpic', () => {
 	it('returns an api.post(query) action', () => {
 		return postEpic(MOCK_POST_ACTION).then(([action]) => {
 			const { query, onSuccess, onError } = MOCK_POST_ACTION.payload;
-			const expectedAction = api.del(query, { onSuccess, onError });
+			const expectedAction = api.post(query, { onSuccess, onError });
```

Comment:
Just want to validate this is intentional. Makes sense based on test description though

Reply:
yeah, I'm sort of surprised that this wasn't throwing an error previously, but maybe there was something else not working right